src/model/model.nit

```diff
@@ -1,6 +1,10 @@
 # This file is part of NIT ( http://www.nitlanguage.org ).
 #
-# Copyright 2012 Jean Privat <jean@pryen.org>
+# Copyright 2012-2016 Jean Privat <jean@pryen.org>
+# Copyright 2013-2016 Alexandre Terrasa <alexandre@moz-code.org>
+# Copyright 2014-2015 Lucas Bajolet <r4pass@hotmail.com>
+# Copyright 2014-2015 Alexis Laferrière <alexis.laf@xymus.net>
+# Copyright 2016      Jean-Christophe Beaupré <jcbrinfo@users.noreply.qithub.com>
```

Comment:
Please do not touch Copyright. We stopped to use them since they are greatly unreliable (and Git is sufficient)